src/onegov/swissvotes/layouts/default.py

```diff
@@ -154,24 +155,23 @@ def format_policy_areas(self, vote: 'SwissVote') -> str:
         for path in (area.label_path for area in vote.policy_areas):
             paths.setdefault(path[0], []).append(path)
 
-        # FIXME: use Markup
         translate = self.request.translate
-        return ",<br>".join(
-            "<span title=\"{}\">{}</span>".format(
-                " &#10;&#10;".join(
-                    " &gt; ".join(translate(part) for part in title)
-                    for title in titles
-                ),
-                translate(value)
+        return Markup(  # noqa: MS001
+            ",<br>".join(
+                "<span title=\"{}\">{}</span>".format(
+                    " &#10;&#10;".join(
+                        " &gt; ".join(translate(part) for part in title)
+                        for title in titles
+                    ),
+                    translate(value)
+                )
+                for value, titles in paths.items()
```

Comment:
```suggestion
        return Markup(",<br>").join(
               Markup( "<span title=\"{}\">{}</span>").format(
                    Markup(" &#10;&#10;").join(
                        Markup(" &gt; ").join(translate(part) for part in title)
                        for title in titles
                    ),
                    translate(value)
                )
                for value, titles in paths.items()
```
Ansonstent bringt es nicht viel. Es sei denn die einzelnen parts dürfen effektiv Markup enthalten.